features/DeviceShadow.feature.md

```diff
@@ -343,11 +341,3 @@ Then the response should equal to this JSON
   }
 }
 ```
-
-## Verify I will not receive the device shadow if the version is not updated
```

Comment:
Removed this because this is not the behavior of the system. It does not send a blank message.